Culture handbook updates (#10241)
PostHog/posthog.com: contents/handbook/company/culture.md

So, what's it like working at PostHog?

## We're 100% remote 

> [Our team](/people) is 100% remote, and distributed across more than 15 countries.

Reviewer: @smallbrownbike we should automate this number

Reviewer 2: There's some hardcoded reference in [contents/handbook/company/grown-ups.md](https://github.com/posthog/posthog.com/tree/master/contents/handbook/company/grown-ups.md) as well

> In addition to all the equipment you'll need, we provide [a budget to help you find coworking space](/handbook/people/spending-money#work-space), or to cover the costs of coffees for those who prefer not to work at home.

Reviewer 3: I think this should be below the list of advantages.

Author: done

Being all remote has a bunch of advantages:

* We can hire [amazing people](/people) from a global talent pool.
* It encourages thoughtful, and intentional, written communication
* It creates space for lots of uninterrupted work.

> * We judge performance based on real outcomes, not hours spend in the office.

Reviewer: "in the office" → "in front of your computer"?

Author: Could change to "hours spent in an office". I think the point is begin remote eliminates presenteeism, which is specifically associated with being in an office.

> * We judge performance based on real outcomes, not hours spend in the office.

Author: ```suggestion
* We judge performance based on real outcomes, not hours spent in the office.
```

Putting things in writing helps us clarify our own ideas, as well as allow others to provide better feedback. It has been key to our development and growth.

> ## We don't let others fail

Reviewer 4: I feel like this is somewhat at odds with our performance expectations and the fact that we do let people go if they aren't able to perform.

Reviewer 4: Yeah I feel like this heading has nothing to do with the content. Maybe "We give direct feedback early and often" and link to the blog post about this?

> ## We don't let others fail

Author: ```suggestion
## We give direct feedback early and often
```

Everyone should help everyone else raise their game. Fatigue sets in when you complete a task, so it's easier for outsiders to help those creating the work to raise their game.

We are direct about the quality of work. That doesn't always mean work needs to be completely polished, as it depends on the speed and impact of a task. Being great at [giving and receiving feedback](/handbook/people/feedback) is a key part of our culture. 

> ## We bias for action

Reviewer 5: yes! have wanted to switch that back for ages

Putting things in writing helps us clarify our own ideas, as well as allow others to provide better feedback. It has been key to our development and growth.

## We don't let others fail

Everyone should help everyone else raise their game. Fatigue sets in when you complete a task, so it's easier for outsiders to help those creating the work to raise their game.

We are direct about the quality of work. That doesn't always mean work needs to be completely polished, as it depends on the speed and impact of a task. Being great at [giving and receiving feedback](/handbook/people/feedback) is a key part of our culture. 

## We bias for action

> If given a choice, go live. If you can't go live, reduce the task size, so you can.

Reviewer: feels like an unnecessary comma ;)

> If given a choice, go live. If you can't go live, reduce the task size, so you can.

Author: ```suggestion
If given a choice, go live. If you can't go live, reduce the task size so you can.
```

Default to _not_ asking for permission to do something if you are acting in the best interests of PostHog. It is ok to ask for more context though. 

> ## We on the maker's schedule

Reviewer 4: ```suggestion
## We're on the maker's schedule
```

We're big believers in the importance of the [maker's schedule](http://www.paulgraham.com/makersschedule.html). If we have meetings at all, we'll cluster them around any stand-ups, so our day doesn't get split up.

![Screenshot of an engineer's calendar at PostHog](https://res.cloudinary.com/dmukukwp6/image/upload/a0d634a2_bd3e_4229_ae8f_f98269a6c4f7_2268x1473_06595e2e80.jpg)

On Tuesdays and Thursdays, we don't have internal meetings at all. Occasionally an external meeting will slip in on those days, such as interviews, but we try to keep those to an absolute minimum.

## We're structured for speed and autonomy

Hiring high-performing and self-sufficient team members means we don't need the typical corporate processes that are designed to slow teams down. Instead, we're organized into [small teams](/handbook/team-structure), which prioritize speed by delegating decision-making autonomy as much as possible. 

> Our [management approach](/handbook/company/management) is super simple - James Hawkins, Tim, and the team leaders are the only managers, and everyone else reports to one of them. We don't want to create a fancy hierarchy of titles, as we believe this can lead, consciously or not, to people feeling less empowered to make changes and step on toes, especially if they are not in a 'senior' role. 

Reviewer 5: ```suggestion
Our [management approach](/handbook/company/management) is super simple - small teams report to their team leader, and each of the team leaders reports to one of our four execs. We don't want to create a fancy hierarchy of titles, as we believe this can lead, consciously or not, to people feeling less empowered to make changes and step on toes, especially if they are not in a 'senior' role. 
```

> It's up to you how to get things done. If you want to make a change, feel free to just create the pull request. If you want to discuss something more widely for a bigger piece of work, it might make sense to use an RFC for a change inside your team. 

Reviewer 3: ```suggestion
It's up to you how to get things done. If you want to make a change, feel free to just create the pull request. If you want to discuss something more widely for a bigger piece of work, it might make sense to [use an RFC](/handbook/company/communication#requests-for-comment-rfcs) for a change inside your team. 
```
